### PR TITLE
Rewrite the development setup with all the needed steps

### PR DESCRIPTION
**Pull request by rafaels88**
This PR is a proposition for rewriting the `Local development` section with some missing steps you need to take in order to set up the project locally.

All these steps were taken based on my own experience for setting up the project.

Something the annoyed me is that I am not sure what is the minimum version for Postgresql. I am keen to change the PR to add this info.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,17 +4,43 @@ Source for https://api.ooni.io/
...
+* Make
+* Python >= 3.5
+* Postgresql
 
-```
-make serve
+### Quickstart
+
+**Note**: the default database configuration is `postgres@localhost:5432/ooni_measurements`,
+you only need to run the second step below in case you want to use a different one.
```
Comment (bassosimone) [inline Readme.md]: What is the second step? `export DATABASE_URL`? I'd suggest to say that explicitly.

**Comment by hellais** [comment]
Thanks for working on this!

You can also drop from the documentation reference to `docker-compose` as it was dropped since: https://github.com/ooni/api/commit/faec3c451391f464c5b2127f74f879cb461fe5b4

File: Readme.md (hunk 1 above)
Comment (rafaels88) [reply Readme.md]: Make sense, gonna change it!

**Comment by rafaels88** [comment]
@hellais gonna do that!

File: Readme.md (hunk 1 above)
Comment (rafaels88) [reply Readme.md]: @bassosimone Added it into the paragraph, see if it's any better now

**Comment by rafaels88** [comment]
@hellais Removed Docker Compose as a requirement, take a look if I missed something

File: Readme.md (hunk 1 above)
Comment (bassosimone) [reply Readme.md]: thanks

**Comment by bassosimone** [comment]
(I see that tests are broken but the breakage is unrelated to your PR and arguably predates it, hence I am going to merge nonetheless and open an issue about broken tests.)

**Pull request merged**